client/dashboard/data/site-jetpack-monitor-uptime.ts

```diff
@@ -5,15 +5,32 @@ export interface SiteUptime {
 	downtime_in_minutes?: number;
 }
 
-export async function fetchSiteUptime(
-	siteId: number,
-	period?: string
-): Promise< Record< string, SiteUptime > | undefined > {
-	return wpcom.req.get(
+export async function fetchSiteUptime( siteId: number, period?: string ): Promise< number | null > {
+	const uptime: Record< string, SiteUptime > = await wpcom.req.get(
 		{
 			path: `/sites/${ siteId }/jetpack-monitor-uptime`,
 			apiNamespace: 'wpcom/v2',
 		},
 		{ period: period ?? '30 days' }
 	);
+
+	// Post-process the data to calculate total number for up and down days during the time period.
+	const { upDays, downDays } = Object.values( uptime ).reduce(
```

Comment:
I think it would be better to put it in the `select` of the react-query 🤔

Reply:
Oh yes it should be in select. 0580ff83c53a25ae11dfbcc1ab4a0d895ad9526c.